src/backend/rest.ts

```diff
@@ -66,7 +66,7 @@ export default class RestApiBackend implements Backend {
   }
 
   async testDatasource(): Promise<unknown> {
-    const result = await this.api<{versions: {checkmk: string}; edition: string}>({
+    const result = await this.api<{ versions: { checkmk: string }; edition: string }>({
```

Comment:
please remove the "fix formatting" commit and fix the formatting in those commits that break the formatting.